src/contact-picker/contact-picker.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { TestBed, ComponentFixture } from '@angular/core/testing';
 import { ElementRef } from '@angular/core';
 
 import { ContactPickerComponent, ContactPickerService, ContactPickerValue, ContactPickerModule } from '..';
-import { Observable } from 'rxjs/Observable';
+import { of as observableOf } from 'rxjs';

```

Comment:
Gewoon uit curieusiteit (omdat het niet nodig is): waarom hanteer je hier `as observableOf`?

Reply:
Voor leesbaarheid.
Anders krijg je in de code gewoon `...return of(dataSource...` wat niet meteen duidelijk maakt wat precies doet (zijnde er een Observable van maken).
Ik denk trouwens dat ik dat ergens was tegengekomen in de RxJS documentatie.